ovos_date_parser/dates_da.py

```
@@ -781,3 +781,69 @@ def date_found():
+    for unit in time_units:
+        unit_nl_words = da_translations[unit]
+        unit_nl_words.sort(key=len, reverse=True)
+        for unit_nl in unit_nl_words:
+            unit_pattern = pattern.format(unit=unit_nl)
+            matches = re.findall(unit_pattern, text)
+            value = sum(map(float, matches))
+            time_units[unit] = time_units[unit] + value
+            text = re.sub(unit_pattern, '', text)
+
+    text = text.strip()
+    duration = timedelta(**time_units) if any(time_units.values()) else None
```

Ruff (0.8.2)

Undefined name `timedelta`

(F821)